server/types/oid/iterate.go

```diff
@@ -375,14 +441,19 @@ func iterateIndexes(ctx *sql.Context, callbacks Callbacks, itemSchema ItemSchema
 // the relevant callback is called with the item. This means that, at most, only one callback will be called. If the
 // item cannot be found, then no callbacks are called.
 func RunCallback(ctx *sql.Context, oid uint32, callbacks Callbacks) error {
+	if callbacks.Type != nil {
+		return runType(ctx, oid, callbacks)
+	}
```

Comment:
This should go after the validation, and should check that the section matches either `BuiltIn` or `Section_Type`.